src/System.Linq/src/System/Linq/First.cs

```diff
@@ -34,17 +32,11 @@ public static TSource First<TSource>(this IEnumerable<TSource> source, Func<TSou
             return first;
         }
 
-        public static TSource FirstOrDefault<TSource>(this IEnumerable<TSource> source)
-        {
-            bool found;
-            return source.TryGetFirst(out found);
-        }
+        public static TSource FirstOrDefault<TSource>(this IEnumerable<TSource> source) =>
+            source.TryGetFirst(out bool found);
```

Comment:
I quite like naming unused out variables like this `_` to reflect their unimportance to the given use. If other people also find this useful I could apply that here, but I don't recall anything suggesting this in the style guide.